### ISSUE TITLE
Tooltip present in exported PNG/JPG/PDF

### ISSUE THREAD

**Issue by maurycyg** [amcharts/amcharts4#568]
Expected behaviour - exporting just the drawn chart with axis/labels
Currently it does export tooltips as well (see attached exported PNG)

- Issue found in 4.0.0-beta.81
- Issue not present in 4.0.0-beta.70

![firstchart 4](https://user-images.githubusercontent.com/701197/48571893-ed47a480-e8ff-11e8-80f4-e1fe3f682505.png)

**Comment by maurycyg** [comment]
I'm using `chart.exporting.export(fileType)`, that did work correctly in the previous version and did hide the tooltips

**Comment by maurycyg** [comment]
I think this should be an option in export instead

**Comment by martynasma** [comment]
Defaults are tricky and very subjective. My personal opinion is that chart should be exported as it looks in the present moment.

Creating a setting for tooltip exporting would mean we need to create settings for any possible other chart interactions. For example, should we zoom the chart out before exporting? Should we reset hover states as well? Or pull back in pulled Pie chart slices? What about scrollbars: should they be exported or not?

We'd be opening up the whole new Pandora's box of complexity and bogging down the API with a bunch of additional settings.

I hope my point makes sense.

**Comment by martynasma** [comment]
Good news.

So, I slept on it for a bit, and was able to implement an elegant solution: `exportable` property on any element.

This is still in dev until new version is released, but once it's out you will be able to to disable tooltips in export easily like this:

```
series.tooltip.exportable = false;
```

This works on virtually any element. For example if you don't want scrollbars to appear in export, you do:

```
chart.scrollbarX.exportable = false;
```

Scrollbar grips and zoom out button are set `exportable = false` by default.

Hope you like this approach.

I expect the new version to be released within 1-2 days. I'll ping you here once its out.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.84](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.84).

## [4.0.0-beta.84] - 2018-11-18

### Added
- `exportable` (default `true`) property added to `Sprite`. If set to `true` this element will not appear in the exported image of the chart.

### Changed
- `exportable` is now set to `false` by default on `Scrollbar` grips, zoom out button.

### Fixed
- JSON: `dateFormats` was being ignored.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by hshahul** [comment]
I tried    "exportable": false, for cursor, tooltip also, but not working.. I am using the latest version of amChartjs. can you give any solution?
